en/firefox/features/independent.ftl

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/features/independent/
+
+# HTML page title
+features-independent-firefox-a-different = { -brand-name-firefox }, a different browser for different times. Browse free.
+
+# HTML page description
+features-independent-browse-the-internet = Browse the Internet as it was meant to be… free, safe and accessible to all. Declare your online independence.
+features-independent-firefox-rebel-with-a = { -brand-name-firefox }: Rebel with a cause
+features-independent-firefox-is-independent = { -brand-name-firefox } is independent and a part of the non-profit { -brand-name-mozilla }, which fights for your online rights, keeps corporate powers in check and makes the Internet accessible to everyone, everywhere.
+
+# This phrase means without conditions or restrictions, expecting nothing in return
+features-independent-no-strings-attached = No strings attached
+features-independent-firefox-is-built-by = { -brand-name-firefox } is built by a non-profit. That means we can do things that others can’t, like build new products and features without a hidden agenda. We champion your right to privacy with tools like Private Browsing with Tracking Protection.
+
+# Obsolete string
+features-independent-firefox-is-built-by-old = { -brand-name-firefox } is built by a non-profit. That means we can do things that others can’t, like build new products and features without a hidden agenda. We champion your right to privacy with tools like Private Browsing with Tracking Protection, which go beyond what { -brand-name-google } { -brand-name-chrome } and { -brand-name-microsoft } { -brand-name-edge } offer.
```

Comment:
Let's standardize how we distinguish the obsolete and the updated string in the string ID. I have seen `old`, `fallback`, and `update`. The first two label the obsolete strings while the latter marks the revised string.